package/main/daml/Daml.Finance.Claims/daml.yaml

```diff
@@ -4,21 +4,21 @@
 sdk-version: 2.7.0
 name: daml-finance-claims
 source: daml
-version: 2.0.99.20231115.1
+version: 2.0.100
```

Comment:
In our versioning, `2.0.99.xyz` is a pre-release for `2.1.0`, not `2.0.100`. Another point: are we switching to non-prerelease versions with this PR already, would this be something we do closer to the RC date?